### DISCUSSION TITLE
module 'ffmpeg' has no attribute 'Error'

### DISCUSSION THREAD

**Discussion by phil123456** [openai/whisper#487] [General]
Hi,

how do I solve this ?

**module 'ffmpeg' has no attribute 'Error'**

I just did pip install ffmpeg after having installed the repo with pip

the error occurs in the latest jupyter-lab (fresh install)

thanks

**Comment by RhombusWeasel** [comment]
I think you need to install the ffmpeg-python package as referenced [here](https://github.com/kkroening/ffmpeg-python/issues/174)

**Comment by phil123456** [comment]
nop

apparently it is already installed

Requirement already satisfied: ffmpeg-python in c:\users\phil\appdata\local\programs\python\python39\lib\site-packages (0.2.0)
Requirement already satisfied: future in c:\users\phil\appdata\local\programs\python\python39\lib\site-packages (from ffmpeg-python) (0.18.2)

**Comment by phil123456** [comment]
ok I did this instead

pip uninstall ffmpeg
pip uninstall ffmpeg-python

and install ffmpeg-python again with :
pip install ffmpeg-python

**Reply by Bassell-Iddisah** [reply]
hi, i just did what you recommended but im still getting the file not found error

**Reply by souravsingh0990** [reply]
same here
